app/Resources/views/base/admin/main_navigation.html.twig

```diff
@@ -318,9 +318,16 @@
                 ]
             } %}
 
-            {% if is_granted_admin() %}
-                <li class="nav-title">Admin</li>
-                {% include 'base/admin/nav_link.html.twig' with {
+			<li class="nav-item">
+				<a class="nav-link" href="https://vektorprogrammet.slab.com/">
+					<i class="fa nav-icon"><img src="{{ absolute_url(asset('images/slab.svg')) }}" alt="https://vektorprogrammet.slab.com/"></i>
+					Slab
+				</a>
+			</li>
+
+			{% if is_granted_admin() %}
+				<li class="nav-title">Admin</li>
+				{% include 'base/admin/nav_link.html.twig' with {
```

Comment:
The indent is off. pls fix.

Otherwise, 10/10! Great way to make it easier for people to use (and remember) Slab!